modules/backend/lang/fr/lang.php

```diff
@@ -29,9 +35,24 @@
         ],
```

Comment:
```suggestion
        ],
        'invalid_token' => [
            'label' => 'La clé de sécurité est invalide',
        ],
```

Reply:
@mjauvin Why do you want to add this entry?
It doesn't exist in the EN file.